src/main/java/com/google/cloud/spanner/pgadapter/ConnectionHandler.java

```diff
@@ -220,6 +220,10 @@ public synchronized void addActiveStatement(IntermediateStatement statement) {
     activeStatementsMap.put(this.connectionId, statement);
   }
 
+  public synchronized IntermediateStatement getActiveStatement() {
```

Comment:
Could you elaborate what the active statement means?

Also, a general comment/question: I believe when we receive a QueryMessage with the COPY command inside, then PGAdapter should be set up to expect messsages relevant to COPY, and return errors if it receive unexpected messages. Does the current PGAdapter implementation already handle this? If not, maybe we should mention it in our design doc, and come up with a mechanism to do it.